views/sign-out.html

```diff
@@ -6,6 +6,9 @@
 {% endif %}
 
 {% block pageTitle %}
+  {% if errors and errors.errorList and errors.errorList.length > 0 %}
+  Error:
```

Comment:
You could indent 'Error:' by a couple of spaces for readability as it's in a block.

Assume you've tested this change in the app with Docker?